.github/workflows/ci.yml

```diff
@@ -70,6 +70,72 @@ jobs:
           name: crash-logs-${{ matrix.os }}-java-${{ matrix.java }}
           path: ${{ steps.copy_test_logs.outputs.dir }}
 
+  java-topology-build:
+    name: Java Topology Tests (JDK ${{ matrix.java }}, ${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        java: [ '8' ]
+        os: [ 'ubuntu-20.04' ]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Cache Gradle dependencies
+        uses: actions/cache@v2
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-caches-${{ hashFiles('**/*.gradle') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-caches-
+      - name: Cache Gradle wrappers
+        uses: actions/cache@v2
+        with:
+          path: ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-wrapper-${{ hashFiles('**/gradle/wrapper/gradle-wrapper.properties') }}
+      - name: Setup java
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+      - name: Setup BUILD_JAVA_HOME (Linux)
+        if: runner.os == 'Linux'
+        run: echo "BUILD_JAVA_HOME=${JAVA_HOME}" >> $GITHUB_ENV
+      - name: Setup java 8 to run the Gradle script
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - name: Setup virtual network interfaces
+        run: ./aeron-samples/scripts/cluster/setup-namespaces
+      - name: Build aeron with Gradle
+        run: ./gradlew -x javadoc -x test
```

Comment:
I would simply go for:
```
./gradlew clean assemble -x javadoc
```
This way it will be independent of the default commands in the `build.gradle` file.